clients/repo_client.go

```diff
@@ -40,4 +40,5 @@ func NewRepoUnavailableError(err error) error {
 type RepoClient interface {
 	InitRepo(owner, repo string) error
 	GetRepoArchiveReader() (io.ReadCloser, error)
+	ReleaseRepo() error
```

Comment:
Is there a reason to add this API? If not, I would prefer keeping this as is.

Reply:
If the reason for having `ReleaseRepo` is to delete the tarball file after use, then let's call this method `Close` instead. It will be inline with other common Golang APIs.

And instead of calling `Close` at the end of every `RepoInit`, we simply call it once after we are done using the `RepoClient`. For example, add `defer repoClient` here - https://github.com/ossf/scorecard/blob/main/cron/worker/main.go#L174

Reply:
> If the reason for having `ReleaseRepo` is to delete the tarball file after use, then let's call this method `Close` instead. It will be inline with other common Golang APIs.

other golang APIs use `Open` and `Close`, but ours is called `InitRepo`. What's the opposite of `init`? I could not find a better word.

> And instead of calling `Close` at the end of every `RepoInit`, we simply call it once after we are done using the `RepoClient`. For example, add `defer repoClient` here - https://github.com/ossf/scorecard/blob/main/cron/worker/main.go#L174

Note sure I follow. I currently call `defer repoClient.ReleaseRepo()` after calling `InitRepo()`, see https://github.com/ossf/scorecard/pull/598/files#diff-afbd0f7e33a55279ebab362790883a75392a3f020c4c7d2de49e8322f08e2317R89.